recipes/certificate.rb

```diff
@@ -26,11 +26,26 @@
     key_path  node['abiquo']['certificate']['key_file']
     not_if { ::File.file? "node['abiquo']['certificate']['file']" }
     only_if { node['abiquo']['certificate']['source'] == 'self-signed' }
-    notifies :restart, 'service[apache2]' unless node['abiquo']['profile'] == 'websockify'
+    notifies :restart, 'service[apache2]' if node.recipe?('abiquo::install_ui')
     notifies :import, "java_management_truststore_certificate[#{node['abiquo']['certificate']['common_name']}]", :immediately
     notifies :create, "template[#{node['abiquo']['certificate']['file']}.haproxy.crt]", :immediately
 end
 
+# Collect the API cert for RS
+abiquo_download_cert 'retrieve-api-cert' do
+    host node['abiquo']['properties']['abiquo.server.api.location']
+    notifies :restart, 'service[abiquo-tomcat]' if node.recipe?('abiquo::service')
+    only_if { node['abiquo']['properties']['abiquo.server.api.location'] && node['abiquo']['profile'] == 'remoteservices' }
+end
```

Comment:
Instead of doing this, could we just append the `node['abiquo']['properties']['abiquo.server.api.location']` to the list of additional certs if the profile is `remoteservices` ? That could be already done in the attributes file just after defining the `additional_certs` attribute.